app/MasterTemplate.hs

```diff
@@ -11,6 +11,8 @@ import Config (enableFb, enableCdn)
 import Util.Blaze
 import Scripts (globalScripts)
 
+
```

Comment:
Hey David. I'll make these changed. 

I just wanted to note that I didn't get an email alert for this comment. I think it's because I won't get alerted if you comment on you're review?